src/app/searchResults/searchResults.component.js

```diff
@@ -12,29 +12,52 @@ let componentName = 'searchResults';
 class SearchResultsController {
 
   /* @ngInject */
-  constructor(designationsService) {
+  constructor($window, $location, designationsService) {
+    this.$window = $window;
+    this.$location = $location;
     this.designationsService = designationsService;
+    this.searchParams = {};
   }
 
   $onInit(){
-    let searchTerm = urlQueryTerm.get('q');
-    if(searchTerm){
-      this.designationsService.createSearch(searchTerm)
-        .subscribe((id) => {
-          this.searchId = id;
-          this.getPageResults(1);
+    var params = this.$location.search();
```

Comment:
@Omicron7 found a solution, he committed it directly to master. https://github.com/CruGlobal/give-web/commit/2302727087cdb011a0d6f1cdf308099f81d1dc35